data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -81,40 +82,27 @@ def extract(self) -> None:
         )
 
     def transform(self) -> None:
-        logger.info("Pruning Census GeoJSON")
-        fields = [self.GEOID_FIELD_NAME, self.GEOMETRY_FIELD_NAME]
-        self.geojson_usa_df = self.geojson_usa_df[fields]
-        logger.info("Merging and compressing score CSV with USA GeoJSON")
-        logger.info(f"{self.geojson_usa_df.shape}")
-        self.geojson_score_usa_high = (
-            self.score_usa_df.set_index(self.TRACT_SHORT_FIELD)
-            .merge(
-                self.geojson_usa_df.set_index(self.GEOID_FIELD_NAME),
-                left_index=True,
-                right_index=True,
-                how="left",
-            )
-            .reset_index()
-        )
-
         # Rename GEOID10_TRACT to GEOID10 on score to allow merging with Census GeoJSON
         self.score_usa_df.rename(
             columns={self.TRACT_SHORT_FIELD: self.GEOID_FIELD_NAME},
             inplace=True,
         )
 
+        logger.info("Pruning Census GeoJSON")
+        fields = [self.GEOID_FIELD_NAME, self.GEOMETRY_FIELD_NAME]
+        self.geojson_usa_df = self.geojson_usa_df[fields]
+
         logger.info("Merging and compressing score CSV with USA GeoJSON")
-        geojson_score_usa_high = self.score_usa_df.merge(
+        self.geojson_score_usa_high = self.score_usa_df.merge(
             self.geojson_usa_df, on=self.GEOID_FIELD_NAME, how="left"
         )
-        logger.info(
-            f"{geojson_score_usa_high.equals(self.geojson_score_usa_high)}"
-        )
-        assert 1 == 2
+
         self.geojson_score_usa_high = gpd.GeoDataFrame(
             self.geojson_score_usa_high, crs="EPSG:4326"
         )
 
+        logger.info(f"Columns: {self.geojson_score_usa_high.columns}")
```

Comment:
intentional to keep? i think it could be helpful, maybe just a more descriptive comment.

Reply:
yikes oops,  def not intentional (I pulled this down from main, see comment below, will remove this line :))

Reply:
(fwiw i would like to merge this into main and then separately update this code, because the tests take forever to pass? @lucasmbrown-usds)

Reply:
That's fine!